docs/sources/developers/plugins/metadata.md

```diff
@@ -9,28 +9,128 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
-| info.updated                | Date when this plugin was built. Use `%TODAY%` for Grafana to autopopulate this value.                                        |
-| info.version                | Project version of this commit. Use `%VERSION%` for Grafana to autopopulate this value.                                       |
-
-## Plugin.json Example
-
-Here's an example of an up-to-date plugin.json file:
-
-https://github.com/grafana/clock-panel/blob/master/src/plugin.json
+## Properties
+
+| Property       | Type                    | Required | Description                                                                                                                          |
+|----------------|-------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `dependencies` | [object](#dependencies) | **true** | Plugin dependencies.                                                                                                                 |
+| `id`           | string                  | **true** | Unique name of the plugin. If the plugin is published on Grafana.com then the plugin id has to follow the naming conventions.        |
+| `info`         | [object](#info)         | **true** | Meta data for the plugin. Some fields are used on the plugins page in Grafana and others on Grafana.com if the plugin is published.  |
+| `name`         | string                  | **true** | Human-readable name of the plugin that is shown to the user in the UI.                                                               |
+| `type`         | string                  | **true** | Plugin type. Possible values are `app`, `datasource`, `panel`.                                                                       |
+| `alerting`     | boolean                 | false    | For data source plugins. If the plugin supports alerting.                                                                            |
+| `annotations`  | boolean                 | false    | For data source plugins. If the plugin supports annotation queries.                                                                  |
+| `backend`      | boolean                 | false    | If the plugin has a backend component.                                                                                               |
+| `category`     | string                  | false    | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `sql`.                 |
+| `executable`   | string                  | false    | The first part of the file name of the backend component executables (there can three binaries for linux, darwin and windows).       |
+| `hasQueryHelp` | boolean                 | false    | For data source plugins. TODO not sure if this is used.                                                                              |
```

Comment:
Can be deleted is not used anywhere

Reply:
Should we mark these as deprecated. I ran the schema against all the community plugins, and there's additional fields like `partials`, and `includes`. Maybe we should include them, but with a deprecated flag?

Reply:
Includes is used for embedding dashboards with plugins, right? Partials not sure about, what plugin have that?

Reply:
There are 35 plugins using `partials`, including the [simple-json-datasource](https://github.com/grafana/simple-json-datasource/blob/master/src/plugin.json#L6-L8).

There are 15 plugins using `includes`, including the [kubernetes-app](https://github.com/grafana/kubernetes-app/blob/master/src/plugin.json#L37-L80).

Reply:
Interestingly enough, out of 91 community plugins, only 60 plugins conform this schema.

Majority of the ones that don't conform have the following issues:

- Many are missing `dependencies` or `logos`
- Examples of invalid versions include "^0.0.7", ">=6.3.0", "4.1.1+", "5.0+"
- `updated` has examples of both "2001-01-01", and "01-01-01"